README.md

```diff
@@ -47,6 +67,31 @@ Take note of your api gateway link for later, you can see it in the output under
 npm run dev
 ```
 
+**NOTE:** As of currently you can only look up "National" as that is what is currently in `data-product-kit`'s `query.py`. If you want to change the searchTerm you can do so in the file:
```

Comment:
I feel like this is just pushing the mock data thing we were using further down the pipeline, is there anyway we can use data we've ingested into the local databases?